modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/Diagram.js

```diff
@@ -75,10 +77,53 @@ const Diagram = ({
 		radius: newPinSettings.defaultRadius,
 	});
 
+	const importPinSchema = (svg) => {
+		const textDatas = [];
+		const pinDatas = [];
+		const parser = new DOMParser();
+		const xmlImage = parser.parseFromString(svg, 'image/svg+xml');
```

Comment:
Could we have an empty line between these 2 lines. We want an empty line between variable declaration and its usage.